content/how-to/code-splitting/splitting-css.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to Split CSS?
+contributors:
+  - pksjce
+---
+
+In `webpack`, when you use the css loader and import css into your javascript files, the css is bundled along with your javascript.
```

Comment:
* Just webpack to be consistent with other docs.
* javascript -> JavaScript
* css -> CSS